src/RageUtil.cpp

```diff
@@ -721,6 +721,28 @@ RString join( const RString &sDelimitor, std::vector<RString>::const_iterator be
 	return sRet;
 }
 
+RString serialize(const std::vector<float> & sSource, const RString &sDelimitor, int precision)
+{
+	std::vector<RString> values;

```

Comment:
Same idea here. Can preinitialize the size since we know how many iterations we're doing.